localstack-core/localstack/aws/handlers/openapi.py

```diff
@@ -0,0 +1,71 @@
+"""
+Handlers for validating request and response schema against OpenAPI specs.
+"""
+
+import logging
+
+from openapi_core import OpenAPI
+from openapi_core.contrib.werkzeug import WerkzeugOpenAPIRequest, WerkzeugOpenAPIResponse
+from openapi_core.exceptions import OpenAPIError
+from openapi_core.validation.request.exceptions import (
+    RequestValidationError,
+)
+
+from localstack import config, spec
+from localstack.aws.api import RequestContext
+from localstack.aws.chain import Handler, HandlerChain
+from localstack.constants import INTERNAL_RESOURCE_PATH
+from localstack.http import Response
+
+LOG = logging.getLogger(__name__)
+
+
+class OpenAPIRequestValidator(Handler):
+    """
+    Validates the internal requests with the OpenAPI spec.
```

Comment:
It would be great if we could expand this comment a little bit, as it's pretty short and not 100% clear:
- "internal request" seems to be a bit of an overloaded term. I would think of a request from LocalStack to LocalStack as an internal request (see `InternalRequestParamsEnricher`).
- This module here really should explain why it is necessary.
  - We can globally validate incoming requests, simplifying the implementations of the endpoints.
  - We can use this in testing in order to reconcile that the spec is in sync with the endpoint implementations (I miss static strong typing sometimes 😅).
  - ???